clang/test/SemaSYCL/sycl-no-rdc.cpp

```diff
@@ -1,13 +1,10 @@
 // RUN: %clang_cc1 -fsycl-is-device -verify -fsyntax-only -fno-gpu-rdc -internal-isystem %S/Inputs %s
 
-// Check that declarations of SYCL_EXTERNAL functions throw an error if -fno-gpu-rdc is passed
+// Check that uses of SYCL_EXTERNAL throw an error if -fno-gpu-rdc is passed
 #include "sycl.hpp"
 
-// expected-error@+1{{invalid declaration of SYCL_EXTERNAL function in non-relocatable device code mode}}
-SYCL_EXTERNAL void syclExternalDecl();
```

Comment:
Did we lose this diagnostic for the declaration?

Reply:
nope, before we had a different error for decl vs defn so i was testing both, but now its on the macro use, so defn vs decl doesn't matter and its always the same error.

Reply:
Thanks for explaining.